src/main/scala/where/Condition.scala

```diff
@@ -76,3 +77,12 @@ case class Expression(string: String, references: Reference*) extends Condition
   def identifiables: Set[Identifiable] =
     (references flatMap (_.getReferenceable)).toSet
 }
+
+case class NoRelationshipCondition(node: Node) extends Condition {
```

Comment:
I was actually thinking the same thing! I do think that `HasNoRelationships` would be much nicer to read and reason about